exporter/clickhouseexporter/exporter_logs.go

```diff
@@ -73,28 +73,40 @@ func (e *logsExporter) shutdown(_ context.Context) error {
 
 func (e *logsExporter) pushLogsData(ctx context.Context, ld plog.Logs) error {
 	start := time.Now()
-	err := doWithTx(ctx, e.client, func(tx *sql.Tx) error {
-		statement, err := tx.PrepareContext(ctx, e.insertSQL)
+	err := func() error {
+		scope, err := e.client.Begin()
```

Comment:
this is apparently the way to do batch inserts using the database/sql interface of the clickhouse driver https://clickhouse.com/docs/en/integrations/go/clickhouse-go/database-sql-api#batch-insert